black/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM python:3.7-alpine
+MAINTAINER maintainer=asottile@umich.edu
```

Comment:
I think you mean't `LABEL maintainer=` here? (Ditto throughout.)

Reply:
dangit lol, that's what I get for reading and typing at the same time